### PR TITLE
Remove RPM build

### LINKED ISSUE #296: Linux rpm openssl.so error
### Description

undefined symbol: EC_GROUP_new_curve_GF2m, version OPENSSL_3.0.0 - /usr/lib/pokemon-studio/resources/psdk-binaries/ruby-dist/lib/ruby/3.3.0/x86_64-linux/openssl.so

### Current Behavior:

game not start

### Expected Behavior:

game starting and work

### Steps To Reproduce:

install pokemon studio rpm

<code>sudo dnf -y install https://github.com/PokemonWorkshop/PokemonStudio/releases/download/v2.1.0/pokemon-studio-2.1.0-1.x86_64.rpm</code>

create new game

start not work

### Environment:

Fedora 39 x86_64

### Error return on terminal:

<code>
PSDK_BINARY_PATH="/usr/lib/pokemon-studio/resources/psdk-binaries/" /usr/lib/pokemon-studio/resources/psdk-binaries/ruby-dist/bin/ruby --disable=gems,rubyopt,did_you_mean Game.rb
</code>

<code>
PSDK Version : 26.23
</code>

<code>
An error occured during extensions loading.
</code>

<code>
Error type : LoadError
</code>

<code>
Error message : /usr/lib/pokemon-studio/resources/psdk-binaries/ruby-dist/lib/ruby/3.3.0/x86_64-linux/openssl.so: undefined symbol: EC_GROUP_new_curve_GF2m, version OPENSSL_3.0.0 - /usr/lib/pokemon-studio/resources/psdk-binaries/ruby-dist/lib/ruby/3.3.0/x86_64-linux/openssl.so
/usr/lib/pokemon-studio/resources/psdk-binaries/ruby-dist/lib/ruby/3.3.0/openssl.rb:13:in `require'
/usr/lib/pokemon-studio/resources/psdk-binaries/ruby-dist/lib/ruby/3.3.0/openssl.rb:13:in `<top (required)>'
/usr/lib/pokemon-studio/resources/psdk-binaries/pokemonsdk/scripts/tools/GameLoader/3_load_extensions.rb:14:in `require'
/usr/lib/pokemon-studio/resources/psdk-binaries/pokemonsdk/scripts/tools/GameLoader/3_load_extensions.rb:14:in `<top (required)>'
/usr/lib/pokemon-studio/resources/psdk-binaries/pokemonsdk/scripts/ScriptLoad.rb:194:in `require'
/usr/lib/pokemon-studio/resources/psdk-binaries/pokemonsdk/scripts/ScriptLoad.rb:194:in `load_tool'
/usr/lib/pokemon-studio/resources/psdk-binaries/pokemonsdk/scripts/tools/GameLoader/Z_load_uncompiled.rb:9:in `<top (required)>'
/usr/lib/pokemon-studio/resources/psdk-binaries/pokemonsdk/scripts/ScriptLoad.rb:194:in `require'
/usr/lib/pokemon-studio/resources/psdk-binaries/pokemonsdk/scripts/ScriptLoad.rb:194:in `load_tool'
Game.rb:11:in `<main>'
</code>
<code>
</code>

ps for information on Fedora 39 openssl version 3.1.1 and not 3.0.0

<code>
sudo dnf -y install openssl-devel
Last metadata expiration check: 3:03:50 ago on Sun May 12 12:14:24 2024.
Package openssl-devel-1:3.1.1-4.fc39.x86_64 is already installed.
Dependencies resolved.
Nothing to do.
Complete!
</code>

### PR DESCRIPTION
**Pull request by NuriYuri**
## Description

This PR removes the RPM Build.

Our developper doesn't have magic time they can spend on making thing work on each an every distribution in the world. Especially when users of said distributions are not willing to help. Thus I suggest to remove the support of the whole RedHat ([which is a toxic company anyway](https://www.learnlinux.tv/how-red-hats-open-source-negligence-is-doing-actual-harm-to-the-linux-community-and-enterprise-it-in-general/)) branch. (Like we're not supporting MacOS Intel branch.)

### CODE REVIEW

**Comment by Palbolsky** [comment]
Je ne suis pas contre mais il y a peut-être un entre deux à trouver. Puisque le problème concerne les binaires de psdk et qu'on a pas la capacité de gérer Fedora et co. en plus de tout ce qu'on gère déjà, on pourrait désactiver psdk sur les distributions de Linux autre que Debian, Ubuntu et SteamOS.

**Comment by NuriYuri** [comment]
Si on trouve une condition pour détecter fédora on peut simplement désactiver le play button oui.

**Comment by andykimpe** [comment]
or be careful you 2

you will get a 7 day ban

from the github team

rule it on github and they are very strict about it

it is to speak only and exclusively in English

I myself had posted in French

on a project where the developer was French

although he replied in English

It absolutely doesn't bother her that I write in French

and yet I still got the ban in question

from the github team

They can't all monitor but if they catch you in the act

you will unfortunately have the right to the same ban

at their level that you discuss among yourselves or not

he doesn't really care, he says that the only language allowed on inssu and github pull requests is English

you have the right to do a French-speaking project with a readme in French, they won't tell you anything

but I insist on the fact that the instructions and pull requests must be in English

it's for those who for my part I speak more in French on github

otherwise to answer to disable fedora it is wrong to add the following line in the spec file that you use for creating the rpm

`Conflicts: fedora-release`

but that will in no way solve the problem when I see your last message nuri and I will reply to you afterwards
I understand why it doesn't work on Fedora but I am 90% certain given what you told me that in reality this file is totally incompatible with all RedHat bases

in other words false simply remove the rpm as Pablo says at least for the moment

while I think about the project and provide you with a new packaging

**Comment by NuriYuri** [comment]
We don't have access to the spec file while using Electron Forge. (If you find something inside this: https://js.electronforge.io/interfaces/_electron_forge_maker_rpm.InternalOptions.MakerRpmConfigOptions.html )

Other than that, I'd like to see proof of what you're saying regarding language in PR. I don't find the word english in any of the ToS of Github, if that was mandatory it should be stated somewhere. Other than that, I've seen [PR in Chinese](https://github.com/ant-design/ant-design/pull/47688) and that repository seem to have a [Chinese PR template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md). If other language were forbidden, that wouldn't be possible to have PR templates in other languages. I guess you got reported for another reason ;)

**Comment by andykimpe** [comment]
no I can guarantee you it was good for him

but that was 10 years ago they may have changed the rules since then

I wasn't the only one to be concerned at the time there were a lot of complaints from both project developers and users
